src/components/emissionFactor/new/DetailedGES.tsx

```diff
@@ -45,7 +45,7 @@ const DetailedGES = ({ form, hasParts, setHasParts, partsCount, setPartsCount }:
 
       let totalCo2 = 0
       emissionFactors.forEach((part, index) => {
-        const partTotalCo2 = gazKeys.reduce((acc, gaz) => acc + part[gaz], 0)
+        const partTotalCo2 = gazKeys.filter((key) => !key.endsWith('b')).reduce((acc, gaz) => acc + part[gaz], 0)
```

Comment:
Est ce qu'on peut se dire de mettre un commentaire explicatif de pourquoi on filtre les gaz finissant par 'b' pour les futurs nous ?
Je trouve que c'est non trivial

Reply:
j'en ai aucune idée... C'est comme ca dans la BE :/
Faudrait voir avec Gabriel ?

Reply:
J'étais plutôt juste sur un commentaire dans le code. 
Est ce que tu penses que c'est une bonne pratique ?

Reply:
oui plutot pour comprendre, mais la je sais pas ce que je peux ecrire. Une idée ?